packages/components/src/ColorPicker/useColorPicker.js

```diff
@@ -1,37 +1,157 @@
 import { useContextSystem } from '@wp-g2/context';
-import { cx } from '@wp-g2/styles';
-import { noop } from '@wp-g2/utils';
+import { ui } from '@wp-g2/styles';
+import { noop } from 'lodash';
+import { useCallback, useState } from 'react';
 
-import * as styles from './ColorPicker.styles';
-import { useColorPickerState } from './useColorPickerState';
+export function getColor(color) {
+	return ui.color(color).toRgbString();
+}
```

Comment:
This exported function should be typed:

```suggestion
/**
 * @param {import('tinycolor2').ColorInput} color
 * @return {string}
 */
export function getColor(color) {
	return ui.color(color).toRgbString();
}
```